third_party/mbedtls/mbedtls-config.h

```diff
@@ -120,6 +120,14 @@ __inline int windows_kernel_snprintf(char * s, size_t n, const char * format, ..
 #define MBEDTLS_X509_CRT_PARSE_C
 #endif
 
+#if OPENTHREAD_ENABLE_GCP
```

Comment:
Generally I agree. The reason behind was that in `mbedtls-config.h` there is

`#define MBEDTLS_BASE64_C
`

which is used for JWT rather than ECDSA. Although i can create another configure options if that is preferable.

Reply:
Actually, I was wrong.. `MBEDTLS_BASE64_C` has to be enabled even for ECDSA alone. I'll update PR according to your suggestion, thanks.